lib/views/general/_frontpage_hero.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="homepage-hero">
+  <div class="row">
+    <div class="hero__intro">
+      <h1 class="intro__title"><%= _("Get answers from the government and public sector") %></h1>
+      <p class="intro__description"><%= _("Make a request for information to a UK public authority: by law, they have to respond") %></p>
+        <p class="intro__browse"><%= _(%Q(Browse <a href="#{ request_list_successful_path }">{{number_of_requests}} requests</a> to <a href="#{ list_public_bodies_default_path }">{{number_of_authorities}} authorities</a>),
+    :number_of_requests => number_with_delimiter(InfoRequest.visible.count),
+    :number_of_authorities => number_with_delimiter(PublicBody.visible.count)) %></p>
+    </div>
+    <div class="hero__new-request">
+      <div class="new-request__content">
+        <h2 class="new-request__title"><%= _("Want to know something?") %></h2>
+        <p class="new-request__description"><%= _("Start your own request") %></p>
+        <a class="link_button_green" href="<%= select_authority_path %>"><%= _("Make a request &raquo;") %></a>
+      </div>
+    </div>
+  </div>
+  <div class="hero__learn-more">
+    <div class="row">
+      <div class="learn-more__content">
+        <a href="#learn-more"><%= _("Learn more") %></a>
```

Comment:
I don't think this does anything useful yet. I think we should keep the styling but wait until we have more content on the homepage before we try to link somewhere.